content/en/docs/setup/install/_index.md

```diff
@@ -6,35 +6,72 @@ description: >
   Describes how to install and set up Spinnaker so that it can be configured for use in production.
 ---
 
-This section describes how to install and set up Spinnaker so that it can be configured for
-use in production. If you just want to evaluate Spinnaker without much work, one of the options
-in [Quickstart](/docs/setup/quickstart/) might be a better choice.
+This section describes how to install and set up Spinnaker so that it can be use in production. If
+you just want to evaluate Spinnaker without much work, one of the options in
+[Quickstart](/docs/setup/quickstart/) or a [Local installation](/docs/setup/install/local/) might be
+a better choice.  It's recommended that you review the  [spinnaker architecture](/docs/reference/architecture) 
+to understand how spinnaker operates before installation.
+
+* [Distributed installation](#distributed-installation) on Kubernetes
+
+  Halyard deploys each of Spinnaker's 
+  separately. __This is highly recommended for use in production.__
+
 
 ## What you'll need
 
-* A machine on which to install Halyard
+* A [SQL Databaase](/docs/setup/install/storage/)
+  * MariaDB/MySQL are supported.  This includes various cloud provided databases like AWS RDS & Google's 
```

Comment:
maybe to keep it simpler it should say 
Supports some Cloud providers

I can help by adding a new page that details the cloud providers AWS, Azure, GCP details